test/extended/two_node/tnf_degraded.go

```diff
@@ -0,0 +1,356 @@
+package two_node
+
+import (
+	"context"
+	"encoding/base64"
+	"fmt"
+	"time"
+
+	g "github.com/onsi/ginkgo/v2"
+	o "github.com/onsi/gomega"
+
+	machineconfigv1 "github.com/openshift/api/machineconfiguration/v1"
+	machineconfigclient "github.com/openshift/client-go/machineconfiguration/clientset/versioned"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	policyv1 "k8s.io/api/policy/v1"
+	apierrs "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	k8sruntime "k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/kubernetes"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+const (
+	pdbLabelKey       = "app"
+	pdbLabelValue     = "pdb-demo"
+	pdbDeploymentName = "pdb-demo-deployment"
+	pdbName           = "pdb-demo"
+	rebootTestMCName  = "99-master-tnf-degraded-reboot-block-test"
+	rebootTestMCFile  = "/etc/tnf-degraded-reboot-block-test"
+)
+
+var _ = g.Describe("[sig-apps][OCPFeatureGate:DualReplica][Suite:openshift/two-node] Two Node Fencing behavior in degraded mode", func() {
+	oc := exutil.NewCLI("tnf-degraded").AsAdmin()
+	ctx := context.Background()
+	kubeClient := oc.AdminKubeClient()
+
+	g.BeforeEach(func() {
+		ensureTNFDegradedOrSkip(oc)
+	})
+
+	g.It("PDB minAvailable=1 should allow a single eviction and block the second in TNF degraded mode [apigroup:policy]", func() {
```

Comment:
Lets change this wording to
`should allow a single eviction and block the second in TNF degraded mode when PDB minAvailable=1 [apigroup:policy]`

Also is `TNF degraded mode` a specific status or can we remove the aroncym `TNF` from this test name?